domains/hujjat.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "hujjatiii",
+        "email": "hujjatx@yahoo.com"
+    },
+    "record": {
+        "TXT": "vc-domain-verify=hujjat.is-a.dev,73fd6f83a3b034bfc044"
```

Comment:
TXT **only** dosent really do anything

If you want to connect to your vercel, register `_vercel.hujjat.json` first, then add the TXT

After this, make a new `hujjat.json` that contains a CNAME to one of the vercel server, like `cname.vercel-dns.com`
https://is-a.dev/docs/vercel
```suggestion
```